web/api/health/README.md

```diff
@@ -45,6 +45,7 @@ The following will return an SVG badge of the alarm named `NAME`, attached to th
 ## Health Management API
 
 Netdata v1.12 and beyond provides a command API to control health checks and notifications at runtime. The feature is especially useful for maintenance periods, during which you receive meaningless alarms.
+From Netdata v1.13.1 and beyond, the configuration controlled via the API commands is [persisted across netdata restarts](#persistence).
```

Comment:
if the functionality is becoming available now, i would change this to v1.16 rather.

Reply:
I fixed this with the last commit!